src/style.css

```diff
@@ -2,0 +1,86 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
```

Comment:
You are styling the `body` tag directly. It would be better to add a class to the body and style that class instead. Styling elements using tag names can lead to unexpected results when your CSS grows more complex or when you add more HTML to your page (from the checklist).